lib/arrays.md

```diff
@@ -5,44 +5,65 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is an Array?
 
-Replace me with your response...
+An array is a list of items.
 
 # What are some examples of information that would be Arrays as opposed to some other data type?
 
-Replace me with your response...
+Team rosters would work well in an array, as would an ingredient list.
 
 # What is one way, using Ruby, to retrieve the 6th element in an Array? How about the 8th element? What happens if you try to retrieve the value of the _9999th_ element (or some element that doesn't exist in the array)?
 
-Replace me with your response...
+If your array is simply
+Days = ['Sunday','Monday','Tuesday','Wednesday','Thursday','Friday','Saturday']
+#you can simply
+Days[whatver day you want to call]
```

Comment:
No, you call the array and put the index value in [].  So if you wanted to call Sunday, it is in the '0' index value, you would print days[0].